test/unit/rules/require-mandatory-role-attributes-test.js

```diff
@@ -0,0 +1,128 @@
+import generateRuleTests from '../../helpers/rule-test-harness.js';
+
+generateRuleTests({
+  name: 'require-mandatory-role-attributes',
+
+  config: true,
+
+  good: [
+    '<div role="complementary" />',
+    '<div role="combobox" aria-expanded="false" aria-controls="ctrlId" />',
+    '<div role="option" aria-selected={{false}} />',
+    '<CustomComponent role="checkbox" aria-checked="false" />',
+    '<SomeComponent role={{this.role}} aria-notreal="bar" />',
+    '<OtherComponent @role={{@role}} aria-required={{this.required}} />',
+    '<FakeElement aria-disabled="true" />',
+    '{{some-component role="heading" aria-level="2"}}',
+    '{{foo-component role="button"}}',
+  ],
```

Comment:
We should always have some minimal test cases like the below to ensure we don't break stuff that doesn't use a role at all:

* `<div>`
* `<div foo="true">`
* `<FooComponent>`
* `<FooComponent foo="true">`
* `{{foo-component}}`
* `{{foo-component foo="true"}}`

Reply:
One more tip is to look at the code coverage report output by `npm run test:jest` to ensure every line of the new rule is covered by tests.